Content.Server/Store/Components/StoreComponent.cs

```diff
@@ -66,6 +66,12 @@ public sealed class StoreComponent : Component
     [DataField("buySuccessSound")]
     public SoundSpecifier BuySuccessSound = new SoundPathSpecifier("/Audio/Effects/kaching.ogg");
     #endregion
+
+    /// <summary>
+    /// Is this store a hidden traitor uplink (not nukie), if it is then only traitors can insert currency to prevent valid checking pdas with renault's help.
+    /// </summary>
+    [DataField("traitor")]
+    public bool Traitor;
```

Comment:
the store shouldnt have a notion of traitors, make this a whitelist of roles or something

Reply:
Nevermind, i didnt realize HasRole was VERY old, make this an AntagWhitelist field or something (name due to change) and check if AntagPrototype IDs match

Reply:
dont think todos go in summary so put on its own comment